### PR TITLE
Update cloud-recording.md

### PR DESCRIPTION
**Pull request by GabGonzalezPM**
Updating all the commands needed to fully have a customer self enable meeting recordings in region.
The first edits only included the information for Allow cloud recording; the missing piece is to include the parameter AllowRecordingStorageOutsideRegion

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cloud-recording.md [hunk 1]
```diff
@@ -107,7 +107,14 @@ Set-CsTeamsMeetingPolicy -Identity Global -AllowCloudRecording $false
...
+When this change takes effect, meeting recordings will be stored by default in the nearest geographic region for Microsoft Stream. If your Teams data is stored in-country and you prefer to store meeting recordings in-country, we recommend that you turn off the feature, and then turn it on after Microsoft Stream is deployed to your in-country data residency region. To turn off the feature for all users in your organization, turn off the **Allow cloud recording** setting in the Global Teams meeting policy, which is in the Microsoft Teams admin center. If, however, you still wish to enable recordings to be stored in the nearest geographic region for Microsoft Stream, you must turn on both **Allow cloud recording** and **Allow Recording Storage Outside Region** before this change takes place.
+
+To enable recordings in region the Global policy, use the following cmdlet:
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:112]:
```suggestion
To enable recordings in region in the Global policy, use the following cmdlet:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by GabGonzalezPM** [comment]
@scanum, @tonysmit  Is there a date of when the updates will be live? PArtners are eagerly awaiting this to be public.
Let me know! Thanks

**Pull request merged**